app/controllers/consolidations/DisassociateUCRController.scala

```diff
@@ -29,7 +29,7 @@ import views.html.disassociate_ucr
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class DisassociateUcrController @Inject()(
+class DisassociateUCRController @Inject()(
```

Comment:
Nit: To me it is more readable when acronyms are not capitalised. You still see that it is an acronym by the word itself and it does not get clumped up with the words that follow.

Reply:
I was just trying to make them all consistent

Reply:
If you want to raise a tech debt thing for the board to discuss how we use Acronyms in class/variable names then go for it :)

Reply:
👍